opba-protocols/xs2a-protocol-tests/xs2a-bdd-wiremock/src/main/java/de/adorsys/opba/protocol/xs2a/tests/e2e/wiremock/mocks/MockServers.java

```diff
@@ -152,7 +152,15 @@ public SELF embedded_mock_of_sandbox_for_max_musterman_accounts_running_stateles
 
     public SELF embedded_mock_of_sandbox_for_max_musterman_accounts_running_for_non_happy_path() {
         WireMockConfiguration config = WireMockConfiguration.options().dynamicPort()
-                                               .usingFilesUnderClasspath("mockedsandbox/restrecord-nonhappy/embedded/multi-sca/accounts/sandbox/");
+            .usingFilesUnderClasspath("mockedsandbox/restrecord-nonhappy/embedded/multi-sca/accounts/sandbox/");
+        startWireMock(config);
+
+        return self();
+    }
+
+    public SELF embedded_mock_of_sandbox_for_max_musterman_accounts_running_for_happy_path() {
+        WireMockConfiguration config = WireMockConfiguration.options().dynamicPort()
+            .usingFilesUnderClasspath("mockedsandbox/restrecord/embedded/multi-sca/accounts/sandbox/");
```

Comment:
New fixtures should be placed in completely separate directory - 
`mockedsandbox/restrecord/embedded/multi-sca/accounts-with-balances/sandbox/`
so they do not interfere with existing ones

Reply:
WithBalance is just a new query param. Tests itself pass it or not. There will come other tests for lot with balance in future. Why duplicate all the requests and methods?

Reply:
Tests fixtures MUST be isolated to avoid accidential test pass - if the test does not expect withBalance, there should be no responses for that case at all

Reply:
There is no need to duplicate test or methods - you add special test case and methods for it only. All other tests should remain as intact as it is possible